doc/reference/algorithms/discrete_hausdorff_distance.qbk

```diff
@@ -1,20 +0,0 @@
-[/============================================================================
-  Boost.Geometry
-
-  Copyright (c) 2018 Yaghyavardhan Singh Khangarot, Hyderabad, India.
-  Contributed and/or modified by Yaghyavardhan Singh Khangarot, as part of Google Summer of Code 2018 program.
-
-  Use, modification and distribution is subject to the Boost Software License,
-  Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
-  http://www.boost.org/LICENSE_1_0.txt)
-=============================================================================/]
-
-[def __this_function__ discrete_hausdorff_distance]
-
-
-[heading Behavior]
-The algorithm calculate discrete hausdorff distance between two geometries.
-
-[note The units of the distance depends on strategy. In order to change the
-default behavior a user has to create a strategy and pass it explicitly into
-the algorithm.]
```

Comment:
I'm fairly sure that this should stay and that such description should be there for both algorithms.

Reply:
I have kept this in the updated documentation for both the algorithms.